apps/land/src/widgets/home/ui/OpinionUnit.tsx

```diff
@@ -0,0 +1,61 @@
+import { cn, parse } from "@/shared/utils";
+import { TEXT } from "@/widgets/home/model";
+
+export default function OpinionUnit({
+  reversed,
+  index,
+}: {
+  reversed: boolean;
+  index: number;
+}) {
+  if (reversed) {
+    return (
+      <div className="flex h-fit w-full gap-x-2">
+        <div className="h-38 flex w-full justify-end">
+          <OpinionBox reversed={reversed}>
+            <div>{parse(TEXT.OPINION[index - 1])}</div>
+          </OpinionBox>
+        </div>
+        <Icon index={index} />
+      </div>
+    );
+  }
+  return (
+    <div className="flex h-fit w-full gap-x-2">
+      <Icon index={index} />
+      <div className="h-38 flex w-full justify-start">
+        <OpinionBox>
+          <div>{parse(TEXT.OPINION[index - 1])}</div>
+        </OpinionBox>
+      </div>
+    </div>
+  );
+}
+
+function Icon({ index }: { index: number }) {
```

Comment:
```suggestion
function Icon(index: number) {
```
prop이 한 개면 저는 단순히 이렇게 사용하기도 해요 😃

Reply:
해당 부분은 Icon이 컴포넌트이다 보니 props를 객체 형태로 받아와야 해서 다음과 같이 작성했어요!